cohere/client.py

```diff
@@ -796,3 +800,133 @@ def wait_for_bulk_embed_job(
             timeout=timeout,
             interval=interval,
         )
+
+
+class FinetuneClient:
+    def __init__(self, client: Client):
+        self._client = client
+
+    def get(self, finetune_id: str) -> Finetune:
+        """Get a finetune by id.
+
+        Args:
+            finetune_id (str): finetune id
+        Returns:
+            Finetune: the finetune
+        """
+        json = {"finetuneID": finetune_id}
+        response = self._client._request(f"{cohere.BLOBHEART_URL}/GetFinetune", method="POST", json=json)
+        return Finetune.from_dict(response["finetune"])
+
+    def get_by_name(self, name: str) -> Finetune:
+        """Get a finetune by name.
+
+        Args:
+            name (str): finetune name
+        Returns:
+            Finetune: the finetune
+        """
+        json = {"name": name}
+        response = self._client._request(f"{cohere.BLOBHEART_URL}/GetFinetuneByName", method="POST", json=json)
```

Comment:
find a solution to not access a private member of the client. The same goes for all he other calls.